schema.sql

```diff
@@ -44,4 +44,6 @@ CREATE TABLE visits(
   animal_id INT REFERENCES animals(id),
   vet_id INT REFERENCES vets(id),
   date_of_visit date
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email varchar(120);
```

Comment:
- [ ] Kindly add an index to the following in this file:
- In the `visits` table, the `animal_id` column
- In the `visits` table, the `vet_id` column
- In the `owners` table, the `email` column

Adding indexes to columns sometimes optimises database queries and it is an efficient way to retrieve a relatively small number of rows from a table. Kindly implement this change. 👍 💻

Reply:
@Gambit142, changes done